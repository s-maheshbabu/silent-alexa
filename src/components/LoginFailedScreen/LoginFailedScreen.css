/** Scaffolding **/
#loginfailed-screen {
  grid-column: 1 / 3;
  grid-template-rows: auto;
  height: calc(100vh - 90px);
  overflow: auto;
  text-align: center;
}
