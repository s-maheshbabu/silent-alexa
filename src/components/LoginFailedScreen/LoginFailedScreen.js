import React from "react";

export default function LoginFailedScreen(props) {
  return (
    // TODO
    "Login Failed. This is just a placeholder. We might want to redirect the user to the login page or at least have a link here to let the user try again or go home."
  );
}
