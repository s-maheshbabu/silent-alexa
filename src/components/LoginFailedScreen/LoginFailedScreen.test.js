import React from "react";
import { render, cleanup } from '@testing-library/react';
import LoginFailedScreen from "./LoginFailedScreen";

afterEach(cleanup);

it("renders without crashing", () => {
  const { asFragment } = render(<LoginFailedScreen />);
  expect(asFragment(<LoginFailedScreen />)).toMatchSnapshot();
});
