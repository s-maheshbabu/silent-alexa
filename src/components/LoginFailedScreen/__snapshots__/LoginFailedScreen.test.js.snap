// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<DocumentFragment>
  <div
    id="loginfailed-screen"
  >
    <h1>
       Login Failed 
    </h1>
    <h1>
       This is just a placeholder. We might want to redirect the user to the login page or at least have a link here to let the user try again or go home. 
    </h1>
  </div>
</DocumentFragment>
`;
